services/shhext/chat/encryption.go

```diff
@@ -19,40 +19,51 @@ import (
 
 var ErrSessionNotFound = errors.New("session not found")
 
-// Max number of installations we keep synchronized.
-const maxInstallations = 5
-
 // If we have no bundles, we use a constant so that the message can reach any device.
 const noInstallationID = "none"
 
-// How many consecutive messages can be skipped in the receiving chain.
-const maxSkip = 1000
-
-// Any message with seqNo <= currentSeq - maxKeep will be deleted.
-const maxKeep = 3000
-
-// How many keys do we store in total per session.
-const maxMessageKeysPerSession = 2000
-
 // EncryptionService defines a service that is responsible for the encryption aspect of the protocol.
 type EncryptionService struct {
-	log            log.Logger
-	persistence    PersistenceService
-	installationID string
-	mutex          sync.Mutex
+	log         log.Logger
+	persistence PersistenceService
+	config      EncryptionServiceConfig
+	mutex       sync.Mutex
+}
+
+type EncryptionServiceConfig struct {
+	InstallationID string
+	// Max number of installations we keep synchronized.
+	MaxInstallations uint
```

Comment:
why every other type is regular int? it seems all of them can be only positive. i would keep int for max installation too